cmd/metrics-router.go

```diff
@@ -17,17 +17,16 @@
 package cmd
 
 import (
-	router "github.com/gorilla/mux"
-	"github.com/prometheus/client_golang/prometheus"
+	"github.com/gorilla/mux"
 )
 
 const (
 	prometheusMetricsPath = "/prometheus/metrics"
```

Comment:
IMO, there's no need to prefix with `/prometheus` here.  `/metrics` is the default.